hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeResolverTest.kt

```diff
@@ -39,26 +40,28 @@ class AttributeResolverTest {
 
   companion object {
     @JvmStatic
-    fun testParameters() = Stream.of(
-      of("string", String::class),
-      of("boolean", Boolean::class),
-      of("integer", Integer::class),
-      of("localDate", LocalDate::class),
-      of("localDateTime", LocalDateTime::class),
-      of("list.string", String::class),
-      of("list.boolean", Boolean::class),
-      of("list.integer", Integer::class),
-      of("list.localDate", LocalDate::class),
-      of("list.localDateTime", LocalDateTime::class),
-      of("list.nested.code", String::class),
-      of("list.nested.description", String::class),
-      of("complex.string", String::class),
-      of("complex.boolean", Boolean::class),
-      of("complex.integer", Integer::class),
-      of("complex.localDate", LocalDate::class),
-      of("complex.localDateTime", LocalDateTime::class),
-      of("complex.nested.code", String::class),
-      of("complex.nested.description", String::class),
+    fun testParameters(): Stream<Arguments> = Stream.of(
+      arguments(
```

Comment:
apologies I more meant just replacing each `of` with `arguments` i.e.
```kotlin
      arguments("string", String::class),
      arguments("boolean", Boolean::class),
      arguments("integer", Integer::class),
```

Reply:
I quite like the vararg version of arguments tbh - even less duplication

Reply:
Oh except it doesn't work 😆  Whoops.